README.md

```diff
@@ -15,13 +15,15 @@ Usage: ss-tool [OPTION]...
     -c, --cleanup   removes all git cloned sub-directories & docker db when done
     -g, --git-ref   add an optional custom git reference eg 243 to match issue 243
     -p, --push-git  push to GitHub, default behaviour creates branch but does not push
+    -t, --token     the authorization token to use when pulling from a git repo
         --help      display this help and exit
         --version   display version and exit
 
 
   EXAMPLE(s):
-      ss-tool --cleanup -g 243
-           will remove all git cloned repositories & docker db when done
+      ss-tool --cleanup -g 243 -t $GITHUB_TOKEN
+           will use the github personal access token to clone the repositories
+           then it remove all git cloned repositories & docker db when done
```

Comment:
Small grammar correction possibly needed. Suggest something like:
`then it will remove`   instead of current `then it remove`

Reply:
Thanks.
Resolved here https://github.com/Bank-Builder/ss-tool/pull/5/commits/227e18d735bec8735f192dd8d4709d9f1b9b0d8d